modules/logging-loki-log-access.adoc

```diff
@@ -74,16 +74,36 @@ subjects:
 // tag::CustomAdmin[]
 == Custom admin group access
 
-If you have a large deployment with a number of users who require broader permissions, you can create a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered admins. Admin users have access to all application logs in all namespaces, if they also get assigned the `cluster-logging-application-view` role.
+// tag::LokiMode[]
+If you have a large deployment with several users who require broader permissions, you can create a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered administrators. 
+// end::LokiMode[]
+
+// tag::NetObservMode[]
+If you need to see cluster-wide logs without necessarily being an administrator, or if you already have any group defined that you want to use here, you can specify a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR have the same read access to logs as administrators. 
```

Comment:
I think I'm personally getting a little tripped up by the phrase "or if you already have any group defined that you want to use here", potentially because the phrase is mentioned before the context of what "use here" means (I'm guessing the action of specifying a custom group using the `adminGroup` field).

Would it be possible to remove that phrase from this sentence and make it its own sentence somewhere later in the paragraph?

For example:
```suggestion
If you need to see cluster-wide logs without necessarily being an administrator, you can specify a custom group using the `adminGroup` field. You can also specify any group that is already defined. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR have the same read access to logs as administrators. 
```

Not sure if my suggestion is still technically accurate, but the basis of my suggestion is seeing if the phrase can be moved somewhere after the first sentence.

Reply:
I think your suggestion hangs here, so I'll take it. I agree, the interrupting phrase definitely makes this sentence less clear.